lib/xclarity_client/discover_request_management.rb

```diff
@@ -0,0 +1,32 @@
+require 'json'
+
+module XClarityClient
+  class DiscoverRequestManagement < XClarityBase
+
+    include XClarityClient::ManagementMixin
+
+    def initialize(conf)
+      super(conf, DiscoverRequest::BASE_URI)
+    end
+
+    def discover_manageable_devices(ip_addresses)
+      postReq = JSON.generate([ipAddresses: ip_addresses])
```

Comment:
What pattern of variables name we are using? This variable name should be post_req of post_request. I saw same variables using camelcase, but the pattern used by RedHat is underscore, we need to fallow this.

Reply:
We don't need to follow this pattern, once this client is not exclusive for ManageIQ. But I believe we are following this pattern here in this project, then I agree with you @AndreyMenezes.

Reply:
@AndreyMenezes note that we are using the same pattern that the LXCA REST API uses. But for local variables and variables which aren't related to the endpoint data, I agree using this patter.

Reply:
@walteraa in the beginner we agree to fallow the RedHat pattern for convention, I quickly passed through the code and saw very few variables escaping from the pattern.

Reply:
No, we didn't. We choose the Ruby official convention. Which is using underscore instead of Camelcase. Then, if any Red Hat convention is different than official convention, the Official convention must be the option.